### PR TITLE
Update testing pack to final zip pack

### PR DESCRIPTION
**Pull request by keizer619**

### CODE REVIEW

**Review by pubudu91 — APPROVED** [review]

File: ballerina/build.gradle [hunk 1]
```diff
@@ -895,7 +893,7 @@ task testExamples() {
         def outputFilePath = "${directoryPath}/output.txt"
         def additionalBuildParams = ""
         if (bbe == "secured-service-with-basic-auth" || bbe == "secured-client-with-basic-auth") {
-            additionalBuildParams = "--b7a.config.file=${baseBuildPath}/${bbe}/sample-users.toml"
+            additionalBuildParams = "--b7a.config.file=${distPath}/${bbe}/sample-users.toml"
```
Comment (pubudu91) [inline ballerina/build.gradle:896]: This probably need to be updated right? This is a usage of the old config API. @ldclakmal thoughts?

File: ballerina/build.gradle [hunk 2]
```diff
@@ -1114,7 +1105,7 @@ task testStdlibs() {
         def exitVal
         def additionalBuildParams = ""
         if (stdlibTest == "auth" || stdlibTest == "jwt") {
-            additionalBuildParams = "--b7a.config.file=${baseBuildPath}/${stdlibTest}/tests/resources/datafiles/users.toml"
+            additionalBuildParams = "--b7a.config.file=${distPath}/${stdlibTest}/tests/resources/datafiles/users.toml"
```
Comment (pubudu91) [inline ballerina/build.gradle:1108]: Same here.

File: ballerina/build.gradle (hunk 1 above)
Comment (keizer619) [reply ballerina/build.gradle:896]: Just wondering how these tests were passing, if its an invalid config API shouldn't it fail?

Comment (ldclakmal) [reply ballerina/build.gradle:896]: Yes. We don't need this anymore. These are old config API usages. With the new `configurable` feature, those test cases were also removed. That's why the tests were passing.

Comment (ldclakmal) [inline ballerina/build.gradle:896]:
```suggestion
```

File: ballerina/build.gradle (hunk 2 above)
Comment (ldclakmal) [inline ballerina/build.gradle:1108]:
```suggestion
```

**Pull request merged**